content/en/sheets/guide/logs_analysis.md

```diff
@@ -0,0 +1,90 @@
+5. Click **Save and Open**.
+
+## Adding Calculated Columns
+
+To gain more insight into your return errors, you might want to extract specific parts of the error messages. You can achieve this by adding calculated columns in Sheets.
+
+1. From the header of the far-right column of your table, click the **Plus** icon to add a calculated column.
+2. Use the `REGEXEXTRACT` function to pull out the actual issue with the return. For example, to extract the next word after "Failed" or "Failed to":
+   ```plaintext
+   =REGEXEXTRACT(#'Message', "Failed (?:to )?(\w+)")
+   ```
+   This function helps you identify whether the error is in *getting*, *calculating*, or *handling* returns.
+
+## Using Pivot Tables for Analysis
+
+Pivot tables help you summarize and organize large datasets to find patterns and trends.
+
+1. From an existing spreadsheet that already has a table of data, click **Add Pivot Table**.
+2. In the **Rows** section, select the dimensions you want to analyze, such as the status of logs.
+3. In the **Calculations** section, choose the dimensions for calculations, like sum, average, count, min, and max.
+
+## Example use case: Analyzing retail application error logs
+
+### Problem Statement
+
+You have a retail web application that generates a series of error logs related to "returns." You want to analyze different types of errors causing these issues.
+
+### Example Logs
+
+- `Failed getting returns for the customer!`
+- `Failed to calculate returns for the customer!`
+- `Failed to handle returns for the customer!`
+
+### Steps to Analyze Error Logs
+
+1. **[Start from the Logs Explorer][1]**:
+   - Build a logs query that filters down to Error logs from your retail application with the words “returns” in the log message.
+        ```
+        status:error service:shopist-returns-prod returns
+        ```
+        {{< img src="/sheets/guide/logs_analysis/error_logs_returns.png" alt="Logs Explorer showing the query for Erros Logs from example retail application that contain the word returns in the log message" style="width:80%;" >}}
+
+   - Click **Open in Sheets** to create a new table to analyze these logs.
+        {{< img src="/sheets/guide/logs_analysis/open_in_sheets.png" alt="Modal to add a title to your new spreadsheet from Logs, for example: Returns error analysis" style="width:80%;" >}}
+
+2. **Extract Specific Error Data**:
+   - Add a calculated column in Sheets to pull out the specific issue with the return using the `REGEXEXTRACT` function:
+     ```
+     =REGEXEXTRACT(#'Message', "Failed (?:to )?(\w+)")
+     ```
+
+3. **Analyze Error Types**:
+   - Create a pivot table to count the number of errors per error type (getting, calculating, handling).
+   - Summarize the data to understand the distribution and total impact of each error type.
+        {{< img src="/sheets/guide/logs_analysis/calculated_column_pivot_table.mp4" alt="Video walkthrough of the steps to analyze error logs" video=true >}}
```

Comment:
This looks like it should be a sequence of only numbered steps, not numbered steps with unordered lists under them. It might still make sense to break it into three ordered lists with subheadings (H4s?), but the current formatting implies a lack of ordering that I don't think is correct.